### ISSUE TITLE
Inquiry about yolov5s-VOC.pt

### ISSUE THREAD

**Issue by Wanghe1997** [ultralytics/yolov5#7569]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello author, may I ask which year's VOC dataset is used for the pre-training weight of yolov5s-VOC.pt, the training set and the validation set? thanks

**Comment by zhiqwang** [comment]
Check the following:

https://github.com/ultralytics/yolov5/blob/e54e758cfb4284cde30bcc99bcd8053b0a0a1a1a/data/VOC.yaml#L12-L16

**Comment by Wanghe1997** [comment]
> Check the following:
> 
> https://github.com/ultralytics/yolov5/blob/e54e758cfb4284cde30bcc99bcd8053b0a0a1a1a/data/VOC.yaml#L12-L16

谢谢走哥。所以训练集是这4个一起是吧？

**Comment by zhiqwang** [comment]
哈喽 @Wanghe1997 , 是的, 而且不仅 yolov5 这么做, 似乎所有的 VOC benchmarks 都是采用这样的训练策略.

Yep, not only yolov5 does this, but all VOC benchmarks are trained with this strategy.
